### ISSUE TITLE
PVs don't reattach after power outage

### ISSUE THREAD

**Issue by ahuffman** [jparklab/synology-csi#45]
Sometimes I'll get a power outage and my persistent volumes don't re-attach automatically.  I typically have to kill the csi-attacher pods and synology-csi pods to resolve manually.

Seems like something is probably needed in control loop to reattempt attaching after a failure every X time frame to fix this scenario.

**Comment by reski-rukmantiyo** [comment]
> Sometimes I'll get a power outage and my persistent volumes don't re-attach automatically. I typically have to kill the csi-attacher pods and synology-csi pods to resolve manually.
> 
> Seems like something is probably needed in control loop to reattempt attaching after a failure every X time frame to fix this scenario.

hi @ahuffman , do you mean, have to kill csi-attacher pods and synology-csi pods in order to make it back to normal?

**Comment by ahuffman** [comment]
@reski-rukmantiyo exactly, from what I've observed.  That's the only problem I've ever experienced.
